README.md

```diff
@@ -1,60 +1,8 @@
 # Getting started with CPU Usage on Mbed OS
 
-This guide reviews the steps required to get CPU usage on Mbed OS platform.
+```This example is getting deprecated soon. As this example and "mbed-os-example-cpu-stats" are showing the usage of "mbed_stats_cpu_get" API. Now, "mbed-os-example-cpu-stats" can calculate CPU usage and print with statistics.```
```

Comment:
You can add this paragraph at the top of the file and leave the previous instructions unchanged.
 
A few other points:
- Correct the formatting - use bold formatting instead of triple backticks.
- Link mbed-os-example-cpu-stats when you first mention it and remove "Link to..."
- "This example is deprecated"
- combine the last two sentences